tensorflow_datasets/__init__.py

```diff
@@ -56,7 +56,7 @@
 from tensorflow_datasets import text
 from tensorflow_datasets import translate
 from tensorflow_datasets import video
-
+import tensorflow_datasets as tfds
```

Comment:
What is this line supposed to achieve ? The module cannot import himself.

`import tensorflow_datasets as tfds` should be added to the `cleanup_script.py` script.

Reply:
>I added it in tensorflow_datasets/__init__.py
I am still getting an empty genarator object can you once run and check `iter_dataset_full_names` function. Thanks.

Reply:
I do not see your modifications. Did you correctly imported the `import tensorflow_datasets as tfds` in `cleanup_script.py` ? When you're executing the `cleanup_script.py`, what is the output of `tfds.list_builders()` ?